benchmark/benchmark.c

```diff
@@ -164,143 +196,150 @@ static inline void seL4_BenchmarkTrackDumpSummary(benchmark_track_kernel_entry_t
 
 void notified(microkit_channel ch)
 {
-    switch(ch) {
-        case START:
-            sel4bench_reset_counters();
-            THREAD_MEMORY_RELEASE();
-            sel4bench_start_counters(benchmark_bf);
+    switch (ch) {
+    case START:
+        sel4bench_reset_counters();
+        THREAD_MEMORY_RELEASE();
+        sel4bench_start_counters(benchmark_bf);
 
-            #ifdef CONFIG_BENCHMARK_TRACK_UTILISATION
-            microkit_benchmark_start();
-            #endif
+#ifdef CONFIG_BENCHMARK_TRACK_UTILISATION
```

Comment:
Can we make microkit_benchmark_start() a static inlined noop if CONFIG_BENCHMARK_TRACK_UTILISATION is undefined, and get of of the #ifdef here?